lib/common/commonUtil.py

```diff
@@ -1,31 +1,50 @@
 import os
 import re
 import json
+import numpy as np
 from datetime import datetime
 from environment import Environment
 from logConfig import get_logger
 
 
-def load_json_file(fp):
-    if os.path.exists(fp):
-        with open(fp) as fh:
-            return json.load(fh)
-    else:
-        return {}
-
-
 class StatusRecorder(object):
+    PASS_IMG_COMPARE_RESULT = "PASS_IMG_COMPARE_RESULT"
+
     ERROR_CANT_FIND_STATUS_FILE = "ERROR_CANT_FIND_STATUS_FILE"
     ERROR_FPS_STAT_ABNORMAL = "ERROR_FPS_STAT_ABNORMAL"
     ERROR_ROUND_STAT_ABNORMAL = "ERROR_ROUND_STAT_ABNORMAL"
     ERROR_COMPARING_IMAGE_FAILED = "ERROR_COMPARING_IMAGE_FAILED"
     ERROR_LOOP_TEST_RAISE_EXCEPTION = "ERROR_LOOP_TEST_RAISE_EXCEPTION"
+    ERROR_EVENT_IMAGE_LESS_THAN_2 = "ERROR_EVENT_IMAGE_LESS_THAN_2"
+    ERROR_EVENT_IMAGE_BOTH_SAME = "ERROR_EVENT_IMAGE_BOTH_SAME"
+    ERROR_COMPARE_RESULT_IS_NONE = "ERROR_COMPARE_RESULT_IS_NONE"
+
+    STATUS_IMG_COMPARE_RESULT = "status_img_compare_result"
+    STATUS_SIKULI_RUNNING_VALIDATION = "sikuli_running_stat"
+
+    STATUS_FPS_VALIDATION = "fps_stat"
+    STATUS_FPS_VALIDATION_NORMAL = 0
+    STATUS_FPS_VALIDATION_ABNORMAL = 1
+    TIME_LIST_COUNTER = "time_list_counter"
 
     def __init__(self, status_fp):
         self.status_fp = status_fp
-        self.current_status = load_json_file(status_fp)
+        self.current_status = CommonUtil.load_json_file(status_fp)
+
+    def get_current_status(self):
+        return self.current_status['current_status']
+
+    def record_current_status(self, input_status_dict):
+        self.current_status = CommonUtil.load_json_file(self.status_fp)
+        if 'current_status' in self.current_status:
+            self.current_status['current_status'].update(input_status_dict)
```

Comment:
The same question you came up last time. Should there be a transitional variable for this update()?